contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -751,41 +750,41 @@ TEST(Config, TestHttpOptions) {
   MockApiManagerEnvironmentWithLog env;
 
   static const char config_text[] = R"(
- name: "Service.Name"
- endpoints {
-   name: "Service.Name"
-   allow_cors: true
- }
- http {
-   rules {
-     selector: "ListShelves"
-     get: "/shelves"
-   }
-   rules {
-     selector: "CorsShelves"
-     custom: {
-        kind: "OPTIONS"
-        path: "/shelves"
-     }
-   }
-   rules {
-     selector: "CreateShelf"
-     post: "/shelves"
-   }
-   rules {
-     selector: "GetShelf"
-     get: "/shelves/{shelf}"
-   }
-   rules {
-     selector: "DeleteShelf"
-     delete: "/shelves/{shelf}"
-   }
-   rules {
-     selector: "GetShelfBook"
-     get: "/shelves/{shelf}/books"
-   }
- }
-)";
+    name: "Service.Name"
+    endpoints {
+      name: "Service.Name"
+      allow_cors: true
+    }
+    http {
+      rules {
+        selector: "ListShelves"
+        get: "/shelves"
+      }
+      rules {
+        selector: "CorsShelves"
+        custom: {
+          kind: "OPTIONS"
+          path: "/shelves"
+        }
+      }
+      rules {
+        selector: "CreateShelf"
+        post: "/shelves"
+      }
+      rules {
+        selector: "GetShelf"
```

Comment:
The document does not say anything about required indentation. Please read my previous comment.